ecosystem.json

```diff
@@ -65,6 +65,42 @@
           "address": "time.txryan.com:2002"
         }
       ]
+    },
+    {
+      "name": "sth1.roughtime.netnod.se",
+      "publicKeyType": "ed25519",
+      "publicKey": "9l1JN4HakGnG44yyqyNNCb0HN0XfsysBbnl/kbZoZDc=",
+      "newver": true,
```

Comment:
What information do you want to convey here? That your server supports a newer version? Perhaps the IETF version?

"newver" is not a filed in the config struct. One option is ["version"](https://github.com/cloudflare/roughtime/blob/450b7a11cc4885c4716d8148a927338a646918d2/config/config.go#L31-L34).